Source/WebCore/Configurations/WebCore.xcconfig

```diff
@@ -35,7 +35,8 @@ WK_APPLEJPEGXL_PREPROCESSOR_DEFINITIONS_YES = USE_JPEGXL=1;
 
 // FIXME: remove the interop setting here when rdar://124492036 is resolved.
 WK_HAVE_SWIFT_CPP_INTEROP = HAVE_SWIFT_CPP_INTEROP=0;
-WK_HAVE_SWIFT_CPP_INTEROP[sdk=macos14*] = ;
+WK_HAVE_SWIFT_CPP_INTEROP[sdk=macosx14.4*] = ;
+WK_HAVE_SWIFT_CPP_INTEROP[sdk=macosx14.5*] = ;
```

Comment:
This is unfortunate, because it means we're going to have to update the config for every new `14.x` minor release, otherwise interop will silently be disabled. How about this:
```suggestion
WK_HAVE_SWIFT_CPP_INTEROP[sdk=macosx14*] = ;
WK_HAVE_SWIFT_CPP_INTEROP[sdk=macosx14.0*] = HAVE_SWIFT_CPP_INTEROP=0;
WK_HAVE_SWIFT_CPP_INTEROP[sdk=macosx14.1*] = HAVE_SWIFT_CPP_INTEROP=0;
WK_HAVE_SWIFT_CPP_INTEROP[sdk=macosx14.2*] = HAVE_SWIFT_CPP_INTEROP=0;
WK_HAVE_SWIFT_CPP_INTEROP[sdk=macosx14.3*] = HAVE_SWIFT_CPP_INTEROP=0;
```